packages/editor-ui/src/components/RunDataTable.vue

```diff
@@ -189,6 +201,8 @@ export default mixins(externalHooks).extend({
 			hoveringPath: null as null | string,
 			mappingHintVisible: false,
 			activeRow: null as number | null,
+			columnLimit: 40,
```

Comment:
nitpick - magic number should be set as variable on top `MAX_COLUMNS_LIMIT`

Reply:
so true, please approve again